test/integration/container/multimaster_test.go

```diff
@@ -331,7 +229,8 @@ func TestMultimasterSetup(t *testing.T) {
 
 	// Ensure the local YUM repo is running:
 	if out := runIgnoreError(t, "docker", "inspect", "-f", "'{{.State.Running}}'", "yumrepo"); !strings.Contains(out, "true") {
-		run(t, "docker", "run", "-d", "-p", fmt.Sprintf("%d:80", repositoryPort), "--restart", "always", "--name", "yumrepo", "weaveworks/local-yum-repo:master-7b063513")
+		// NOTE: image must be updated each time Kubernetes or Docker is updated in wksctl
```

Comment:
How do we not forget this?

Reply:
The test will fail since I took out all the upstream references.
This is more a note to help the person trying to figure out what happened.